api/src/main/resources/IdentifierSource.hbm.xml

```diff
@@ -35,6 +35,7 @@
 			<property name="suffix" type="string" />
 			<property name="minLength" type="int" column="min_length" />
             <property name="maxLength" type="int" column="max_length" />
+            <property name="prefixProviderBean" type="string" column="prefix_provider_bean"/>
```

Comment:
Just for clarity to future readers, we might want to refer to 'prefix provider bean _names_' everywhere applicable. Here for instance:
```xml
<property name="prefixProviderBeanName" type="string" column="prefix_provider_bean_name"/>
```